services/tests/e2e/config/plugins/transform-resource-group.js

```diff
@@ -24,6 +24,8 @@ module.exports = {
       newRg.policies.push(wholeRgPolicy);
     }
 
+    newRg.pluginData = { foo: 'bar' };
```

Comment:
`pluginData` should be initialized to an empty object before applying plugins on resource group. So plugins will add some properties to the object rather then replace/create it.